packages/static-hosting/lib/static-hosting.ts

```diff
@@ -571,11 +574,11 @@ export class StaticHosting extends Construct {
       });
     }
 
-    props.behaviourPrefixes?.forEach(prefix => {
+    props.defaultBehaviourPrefixes?.forEach(prefix => {
       additionalBehaviors[`${prefix.prefix}*`] = {
         origin: s3Origin,
         viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
-        edgeLambdas: prefix.edgeLambdas,
+        edgeLambdas: prefix.behaviourOverride.edgeLambdas,
```

Comment:
Instead of just passing the edge lambda you can do this:

```typescript
      additionalBehaviors[`${prefix.prefix}*`] = {
        origin: s3Origin,
        viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
        originRequestPolicy: originRequestPolicy,
        cachePolicy: originCachePolicy,
        responseHeadersPolicy: responseHeadersPolicy,
        ...prefix.behaviourOverride,
      };
```
Because that will then allow you to override any of the properties if you need

Reply:
@finn-holland-aligent this one hasn't been resolved yet